bld/namelist_files/namelist_definition.xml

```diff
@@ -1393,6 +1393,26 @@ scheme (shallow convection).
 Default: set by build-namelist.
 </entry>
 
+<entry id="gw_convect_dp_ml" type="logical" category="gw_drag"
+       group="gw_drag_nl" valid_values="" >
+Whether or not to use the ML scheme for gravity waves produced by deep convection.
+Default: .false.
+</entry>
+
+<entry id="gw_convect_dp_ml_net_path" type="char*132" input_pathname="abs" category="gw_drag"
+       group="gw_drag_nl" valid_values="" >
+Absolute filepath to the deep convection gravity wave neural net used when
+`gw_convect_dp_ml` is set to `.true.`.
+</entry>
+
+<entry id="gw_convect_dp_ml_compare" type="logical" category="gw_drag"
+       group="gw_drag_nl" valid_values="" >
+  Whether or not to run a comparison of the ML deep convection gravity waves to the
+  original scheme. The scheme used to advance the simulation will be dictated by
+  `gw_convect_deep_ml`.
+Default: .false.
+</entry>
```

Comment:
Can we be clearer about what a "comparison" is in this situation as raised by Hamid.
We are not using both schemes, but rather running piggybacking to compare the individual outputs

Reply:
Thanks, I have updated this in https://github.com/DataWaveProject/CAM/pull/4/commits/ac795bb81049a1512c8259262b7be46ffdeb535b